doc/conf.py

```diff
@@ -73,7 +73,6 @@
 # -- General configuration ------------------------------------------------
 
 extensions = [
-    "breathe",
```

Comment:
why was breathe removed?  
Will it affect nordic's documentation team?

Reply:
Breathe was a resource hog for the project, so an alternative has been introduced. We are all aware and already have a working solution that is to serve Doxygen content as upstream does. Btw, documentation CI will be much faster, so no workarounds to make it _fast_ will be needed.